comps/guardrails/toxicity_detection/README.md

```diff
@@ -4,11 +4,13 @@
 
 ## Introduction
 
-Intel also provides toxicity detection model, which is lightweight, runs efficiently on a CPU, and performs well on toxic_chat and jigsaws datasets. More datasets are being fine-tuned. If you're interested, please contact abolfazl.shahbazi@intel.com.
+Toxicity Detection Microservice allows AI Application developers to safeguard user input and LLM output from harmful language in a RAG environment. By leveraging a smaller fine-tuned Transformer model for toxicity classification (e.g. DistilledBERT, RoBERTa, etc.), we maintain a both performant and lightweight guardrails microservice that is readily deployable on both Intel Gaudi and Xeon.
```

Comment:
```suggestion
Toxicity Detection Microservice allows AI Application developers to safeguard user input and LLM output from harmful language in a RAG environment. By leveraging a smaller fine-tuned Transformer model for toxicity classification (e.g. DistilledBERT, RoBERTa, etc.), we maintain a performant and lightweight guardrails microservice that is readily deployable on both Intel Gaudi and Xeon.
```

Reply:
I made the language a little clearer here